src/libraries/System.Net.Security/src/System/Net/Security/SslStream.cs

```diff
@@ -740,12 +740,10 @@ public override int ReadByte()
             }
 
             // Otherwise, fall back to reading a byte via Read, the same way Stream.ReadByte does.
-            // This allocation is unfortunate but should be relatively rare, as it'll only occur once
-            // per buffer fill internally by Read.
-            byte[] oneByte = new byte[1];
-            int bytesRead = Read(oneByte, 0, 1);
+            Span<byte> oneByteSpan = stackalloc byte[1];
```

Comment:
Does this actually improve anything? If this Stream doesn't override Read(Span), and I think it doesn't for a variety of reasons, this will just end up calling the base method which not only has to do the same allocation but also has to do additionsl work, like a copy.

Reply:
After reading your comment, I understand it will degrade performances, will roll back